<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>titlebar</class>
 <widget class="QWidget" name="titlebar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>92</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1140</width>
    <height>92</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1140</width>
    <height>92</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="labelLogo">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>40</y>
     <width>114</width>
     <height>44</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>114</width>
     <height>44</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>114</width>
     <height>44</height>
    </size>
   </property>
   <property name="text">
    <string>logLbl</string>
   </property>
  </widget>
  <widget class="QPushButton" name="settingBtn">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>37</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="customerCenterBtn">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>37</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pBConnect">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>22</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>70</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Not connected</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>50</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string>Centralized Control Center</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBMyPlay">
   <property name="geometry">
    <rect>
     <x>309</x>
     <y>22</y>
     <width>172</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>172</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>172</width>
     <height>70</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>My Performances</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pBMyDownLoad">
   <property name="geometry">
    <rect>
     <x>459</x>
     <y>22</y>
     <width>172</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>172</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>172</width>
     <height>70</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>My Downloads</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pBMyCustom">
   <property name="geometry">
    <rect>
     <x>609</x>
     <y>22</y>
     <width>172</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>172</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>172</width>
     <height>70</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>My Customizations</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="logoSeperatorLbl">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>41</y>
     <width>2</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>2</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>2</width>
     <height>31</height>
    </size>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none; background-color: #B2B9C0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="customerOptBgLbl">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>37</y>
     <width>450</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>450</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>450</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resMainWidget.qrc">:/res/images/img_01.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pBMin">
   <property name="geometry">
    <rect>
     <x>1092</x>
     <y>5</y>
     <width>12</width>
     <height>12</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pBClose">
   <property name="geometry">
    <rect>
     <x>1112</x>
     <y>5</y>
     <width>12</width>
     <height>12</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>pBMin</zorder>
  <zorder>pBClose</zorder>
  <zorder>customerOptBgLbl</zorder>
  <zorder>labelLogo</zorder>
  <zorder>settingBtn</zorder>
  <zorder>customerCenterBtn</zorder>
  <zorder>pBConnect</zorder>
  <zorder>labelTitle</zorder>
  <zorder>pBMyPlay</zorder>
  <zorder>pBMyDownLoad</zorder>
  <zorder>pBMyCustom</zorder>
  <zorder>logoSeperatorLbl</zorder>
 </widget>
 <resources>
  <include location="resMainWidget.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
